### PR TITLE
f-content-cards@v8.0.0 - use newer sass syntax

### PR DESCRIPTION
**Pull request by jamieomaguire**
f-content-cards: v8.0.0
-----------------------------
*June 27, 2022*

### Changed
- **breaking changes** Update to `@use` and `@forward` SASS syntax

Storybook: v0.52.22
------------------------------
*June 27, 2022*
### Changed
- updated vue.config to include f-content-cards in components using @use and @forward

### CODE REVIEW

**Review by FayeCarter — DISMISSED** [review]

File: packages/components/organisms/f-content-cards/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,15 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+
+v8.0.0
+-----------------------------
+*June 27, 2022*
+
+### Changed
+- Update to `@use` and `@forward` SASS syntax
```
Comment (kevinrodrigues) [inline packages/components/organisms/f-content-cards/CHANGELOG.md]: Can we add `**breaking changes** `to this?

Comment (jamieomaguire) [reply packages/components/organisms/f-content-cards/CHANGELOG.md]: Happy to - although I thought major versions were implicitly breaking changes! Will update the readme now :)

Comment (jamieomaguire) [reply packages/components/organisms/f-content-cards/CHANGELOG.md]: updated @kevinrodrigues

**Review by kevinrodrigues — DISMISSED** [review]

**Review by FayeCarter — APPROVED** [review]

File: packages/components/organisms/f-content-cards/CHANGELOG.md (hunk 1 above)
Comment (kevinrodrigues) [reply packages/components/organisms/f-content-cards/CHANGELOG.md]: Yeah agree, i just think it's easier to spot when scrolling through changelogs.

**Review by kevinrodrigues — APPROVED** [review]

**Pull request merged**